modules/nf-core/sourmash/gather/meta.yml

```diff
@@ -0,0 +1,60 @@
+name: "sourmash_gather"
+description: Search the reference database for the most highly contained genomes using sourmash
+keywords:
+  - hash sketch
+  - signature
+  - containment
+tools:
+  - "sourmash":
+      description: Compute and compare MinHash signatures for DNA data sets.
+      homepage: https://sourmash.readthedocs.io/
+      documentation: https://sourmash.readthedocs.io/
+      tool_dev_url: https://github.com/dib-lab/sourmash
+      doi: "10.1186/s13059-016-0997-x"
+      licence: ["BSD-3-clause"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - signature:
+      type: file
+      description: File containing signatures (hash sketches) of a sample
+      pattern: "*.{sig}"
+  - db:
+      type: file
+      description: Sourmash database (list of signatures or SBTs)
```

Comment:
Missing descriptions of new input calues

Reply:
My bad, forgot to add them.
Done now.